parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -74,19 +74,19 @@ downloadCNI() {
     retrycmd_get_tarball 120 5 "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" ${CNI_PLUGINS_URL} || exit $ERR_CNI_DOWNLOAD_TIMEOUT
 }
 
-downloadSecureTLSBootstrapKubeletExecPlugin() {
-    local plugin_url="https://k8sreleases.blob.core.windows.net/aks-tls-bootstrap-client/${SECURE_TLS_BOOTSTRAP_KUBELET_EXEC_PLUGIN_VERSION}/linux/amd64/tls-bootstrap-client"
+downloadSecureTLSBootstrapClient() {
+    local client_url="https://k8sreleases.blob.core.windows.net/aks-tls-bootstrap-client/${SECURE_TLS_BOOTSTRAP_CLIENT_VERSION}/linux/amd64/tls-bootstrap-client"
```

Comment:
This hostname is not currently included in [the list of required egress destinations](https://learn.microsoft.com/en-us/azure/aks/outbound-rules-control-egress) and thus may not be accessible for locked down customers. Can we get it somewhere that we're guaranteed to have access, instead? Perhaps we can package the tool as a container image on MCR and use ctr run with bind mounts to invoke it? This would also let us cache it easily.

Reply:
the plan is to mirror it to acs-mirror once we get that far, which should give us availability in all environments afaik - for now we're just using upstream's staging environment for testing purposes